### ISSUE TITLE
Cannot resolve module 'bootstrap-slider'

### ISSUE THREAD

**Issue by texonidas** [seiyria/angular-bootstrap-slider#132]
Currently attempting to use Webpack to bundle an Angular application, and this error is being emitted. I have http://seiyria.github.io/bootstrap-slider/ installed, however upon closer inspection it doesn't appear to be exported as a module at any point in the code. Am I overlooking something? And should this issue potentially be listed under the bootstrap-slider project?

**Comment by seiyria** [comment]
That's correct, it does not. No, this is an angular bootstrap slider problem. It declares a global module so instead of trying to require it, you can instead just load it (import 'angular-bootstrap-slider' instead of import slider from 'angular-bootstrap-slider'). If that doesn't work, I'm not sure what to tell you.
